tests/accelerators/test_accelerator_connector.py

```diff
@@ -408,12 +415,11 @@ class Accel(Accelerator):
     class Prec(PrecisionPlugin):
         pass
 
-    class TrainTypePlugin(SingleDevicePlugin):
```

Comment:
Why this change?

Reply:
Was just for my own testing purpose. Good spotting :) will change it back